### PR TITLE
Adding support for internal IP for e2e tests

### LINKED ISSUE #50630: Adding support for internal IP for e2e tests
Currently IssueSSHComand in util.go only checks for External IP address
to ssh, this PR adds check for internal IP too.
/sig storage
/kind bug

### PR DESCRIPTION
**Pull request by sbezverk**
Currently IssueSSHComand in util.go only checks for External IP address
to ssh, this PR adds check for internal IP too.
Closes #50630

### CODE REVIEW

**Comment by msau42** [comment]
I don't think these generated files are needed.

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -3219,7 +3219,17 @@ func IssueSSHCommandWithResult(cmd, provider string, node *v1.Node) (*SSHResult,
 	}
 
 	if host == "" {
-		return nil, fmt.Errorf("couldn't find external IP address for node %s", node.Name)
+		// No external IPs were found, let's try to use internal as plan B.
```
Comment (m1093782566) [inline test/e2e/framework/util.go]: Internal IP is marked deprecated?

Comment (sbezverk) [inline test/e2e/framework/util.go]: Could you please provide a reference document about deprecation. I checked this doc and nothing says that node's internal IP is deprecated.

Comment (m1093782566) [inline test/e2e/framework/util.go]: Okay. No issue :)

**Comment by sbezverk** [comment]
Closes #50630

**Pull request merged**